docgen/programming_guide/quickstart.txt

```diff
@@ -30,26 +30,28 @@ Subclass a `Layer`  and define the logic of you program here::
 Always call ``super`` in the constructor::
 
     def __init__(self):
-        super( HelloWorld, self ).__init__()
+        super(HelloWorld, self).__init__()
 
 To display the text, we'll create a `Label`.  Keyword arguments are used
 to set the font, position and alignment of the label::
 
-    label = cocos.text.Label('Hello, world',
-                              font_name='Times New Roman',
-                              font_size=32,
-                              anchor_x='center', anchor_y='center')
+    label = cocos.text.Label(
+        'Hello, world',
+        font_name='Times New Roman',
+        font_size=32,
+        anchor_x='center', anchor_y='center'
```

Comment:
Previous syntax had just one extra character to remove on following lines to be valid.
Also that last line has two arguments that can be on multiple lines IMO.